config/locales/en.yml

```diff
@@ -25,3 +25,178 @@ en:
+              </li>
+              <li>
+                <p>
+                  Open a VNC client and connect to
+                  <code>localhost:%{localport}</code> within the client
+                </p>
+              </li>
+              <li>
+                <p>Use the VNC password: <code>%{password}</code></p>
+              </li>
+            </ol>
+          unsupported: |
+            We do not currently support documentation for connecting to a VNC session 
+            with a native client for your operating system.
+        novnc:
+          launch: "Launch noVNC in New Tab"
+          view_only: "View Only (Share-able Link)"
+        stats:
+          created_at: "Created at:"
+          time_remaining: "Time Remaining:"
+          time_used: "Time Used:"
+          time_requested: "Time Requested:"
+          host: "Host:"
+          undetermined_host: "Undetermined"
+          session_id: "Session ID:"
+          status:
+            starting: "Starting"
+            running: "Running"
+            queued: "Queued"
+            held: "Held"
+            suspended: "Suspended"
+            undetermined: "Undetermined"
+        status_blurb:
+          create_success: "Session was successfully created."
+          delete_success: "Session was successfully deleted."
+          delete_failure: "Session failed to be deleted."
+        delete:
+          title: "Delete"
+          hover: "Delete Session"
+          confirm: "Are you sure?"
+        errors:
+          submission: "Failed to submit session with the following error:"
+          staging: "Failed to stage the template with the following error:"
+      form:
+        launch: "Launch"
+        reset_resolution: "Reset Resolution"
+        session_data: |
+          * All %{title} session data is generated and stored under 
+          the user's home directory in the corresponding %{data_link_tag}.
+        data_root: "data root directory"
+    quota:
+      warning_header: "Quota limit warning for %{path_tag}"
+      reload_message: "Reload page to see updated quota. Quotas are updated every 5 minutes."
+      additional_message: "Consider deleting or archiving files to free up disk space."
+      file: "Using %{used} files of quota %{available} files"
+      file_shared: "(%{used_exclusive} files are yours)"
+      block: "Using %{used} of quota %{available}"
+      block_shared: "(%{used_exclusive} are yours)"
+    sharing:
+      welcome_msg_tag: |
+        Cluster access applications are available from the dropdown menus in the navbar above.
+        <br>Custom shared apps are available below.
+      no_shared_apps_tag: |
+        <strong>No custom shared apps available.</strong>
+      support_msg_tag: |
+        If you don't see the app you are looking for, please
+        <a href="%{support_url}">contact support</a>.
+      catalog_msg_tag: |
+        <br>A subset of the apps available can be viewed on the %{catalog_link_tag}.
```

Comment:
All of these allow actual HTML, so it should instead be `_html` instead of `_tag`. See https://guides.rubyonrails.org/v4.2/i18n.html#using-safe-html-translations.